ci-operator/step-registry/upi/install/gcp/upi-install-gcp-commands.sh

```diff
@@ -0,0 +1,461 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+export HOME=/tmp
+
+export SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+export OPENSHIFT_INSTALL_INVOKER=openshift-internal-ci/${JOB_NAME_SAFE}/${BUILD_ID}
+
+echo "$(date -u --rfc-3339=seconds) - Configuring gcloud..."
+export GOOGLE_CLOUD_KEYFILE_JSON=${CLUSTER_PROFILE_DIR}/gce.json
+gcloud auth activate-service-account --key-file="${GOOGLE_CLOUD_KEYFILE_JSON}"
+gcloud config set project "$(jq -r .gcp.projectID ${SHARED_DIR}/metadata.json)"
+
+echo "$(date -u --rfc-3339=seconds) - Copying config from shared dir..."
+dir=/tmp/installer
+mkdir -p "${dir}/auth/"
+pushd ${dir}
```

Comment:
Another possibly unnecessary `pushd`?

Reply:
`pushd` or `cd`.... either way needs to make PWD for most of the commands in this one.